ee/kafka_client/client.py

```diff
@@ -72,6 +79,14 @@ def produce(self, topic: str, data: Any, key: Any = None, value_serializer: Opti
             key = key.encode("utf-8")
         self.producer.send(topic, value=b)
 
+    def bootstrap_connected(self):
+        """
+        Proxy through to the underlying `KafkaProducer.bootstrap_connected`.
```

Comment:
KafkaConsumer

Reply:
Isn't this going through to the producer though?

Reply:
My bad, this is what happen when you don't click expand the codeblock 😅  <img width="1388" alt="Screenshot 2022-01-25 at 15 21 29" src="https://user-images.githubusercontent.com/4038041/150994338-0402e6df-59fa-446f-a390-d42998196a3e.png">